components/auth/loginPage.js

```diff
@@ -13,21 +13,25 @@ function LoginPage(props) {
       <View style={styles.logo}>
         <Text>LOGO</Text>
       </View>
-      <TextInput style={styles.textInput} placeholder="Email" onChangeText={text => setEmail(text)} />
-      <TextInput style={styles.textInput} placeholder="Password" onChangeText={text => setPassword(text)} />
+      <TextInput style={styles.textInput} placeholder="Email" onChangeText={text => setEmail(text)} autoCapitalize="none" autoCorrect={false} textContentType="emailAddress" />
```

Comment:
Aha I've been needing autoCapitalize='none' + autoCorrect! Is the textContentType a react native feature that form validates?? Will be adding these to my inpuits